build/gulp/tasks/bundle.ts

```diff
@@ -2,70 +2,79 @@ import { task, series, parallel, src, dest } from 'gulp'
 import * as merge2 from 'merge2'
 import * as rimraf from 'rimraf'
 import * as webpack from 'webpack'
+import { argv } from 'yargs'
 
 import config from '../../../config'
+import sh from '../sh'
 
 const { paths } = config
 const g = require('gulp-load-plugins')()
 const { log, PluginError } = g.util
 
+const buildUMD = !!argv.umd
+const packageName: string = argv.package
```

Comment:
lets introduce necessary assertions and provide meaningful error description to the client when this value is not provided (otherwise it might be hard after a while to trace the exact problem why `gulp bundle:build:commonjs` has failed). As an alternative we might introduce default value here (as stardust-react is, essentially, the one that is implied quite often)

Reply:
Let's go with default value 👍